src/Microsoft.AspNetCore.Server.IISIntegration.Tasks/build/netstandard1.0/Microsoft.AspNetCore.Server.IISIntegration.Tasks.targets

```diff
@@ -0,0 +1,35 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <_AspNetCoreIISOldMSBuild Condition=" '$(MSBuildToolsVersion)' == '14.0' Or '$(MSBuildToolsVersion)' == '12.0' ">true</_AspNetCoreIISOldMSBuild>
+
+        <!-- test hook -->
+        <_AspNetCoreIISIntegrationTasksFolder Condition="'$(_AspNetCoreIISIntegrationTasksFolder)' != '' And !HasTrailingSlash('$(_AspNetCoreIISIntegrationTasksFolder)')">$(_AspNetCoreIISIntegrationTasksFolder)\</_AspNetCoreIISIntegrationTasksFolder>
+        <_AspNetCoreIISIntegrationTasksFolder Condition=" '$(_AspNetCoreIISIntegrationTasksFolder)' == ''">$(MSBuildThisDirectory)</_AspNetCoreIISIntegrationTasksFolder>
+    </PropertyGroup>
+
+    <UsingTask AssemblyFile="$(_AspNetCoreIISIntegrationTasksFolder)Microsoft.AspNetCore.Server.IISIntegration.Tasks.dll" TaskName="Microsoft.AspNetCore.Server.IISIntegration.Tasks.TransformWebConfig" />
+
+    <Target Name="PublishForIIS"
+            AfterTargets="Publish"
+            DependsOnTargets="Publish"
```

Comment:
Aren't both are required?
AfterTargets ensures `msbuild /t:Publish` will also run the `PublishForIIS` target.
DependsOnTargets ensures `msbuild /t:PublishForIIS` will do the other `Publish`...open for discussion on this behavior.